src/SQLQueryStress/Form1.cs

```diff
@@ -619,7 +607,7 @@ private void WriteBenchmarkCsvText(TextWriter tw)
                 elapsedTime_textBox.Text,
                 (int)iterations_numericUpDown.Value,
                 (int)threads_numericUpDown.Value,
-                int.Parse(queryDelay_textBox.Text),
+                int.Parse(queryDelay_textBox.Text, CultureInfo.CurrentCulture),

```

Comment:
Invariant?

Reply:
Regarding to the [docs](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1305?view=vs-2019#rule-description), "If the value will be displayed to the user, use the current culture." So I used CurrentCulture for all textbox and csv stuff.

Reply:
csv is not for user display :-)

Reply:
> 
> 
> csv is not for user display :-)

I though it was displayed to user on a CSV file :)

Reply:
Use Invariant - for example to avoid https://stackoverflow.com/questions/2720024/int-parse-of-8-fails-int-parse-always-requires-cultureinfo-invariantculture